marquez_client/client.py

```diff
@@ -306,6 +310,25 @@ def list_datasets(self, namespace_name=None, limit=None, offset=None):
             }
         )
 
+    @staticmethod
+    def make_field(name, data_type, description=None):
```

Comment:
What's the usage of `make_field`?

Reply:
_Minor_: We may want to add this helper method in a `utils.py`, but not a blocker to merge

Reply:
It's mainly used as a helper method when collecting metadata and check for supported field types.
Sounds good! Will move it to a `utils.py` package